retroshare-gui/src/gui/feeds/SecurityIpItem.h

```diff
@@ -60,12 +63,13 @@ private slots:
 	void banIpListChanged(const QString &ipAddress);
 
 private:
-    RsFeedTypeFlags mType;
+	RsFeedTypeFlags mType;
```

Comment:
is that used anywhere?

Reply:
Apparently it is used to discriminate between banned IPs and wrong external IPs